controllers/network/dnsmasq_controller.go

```diff
@@ -445,6 +446,20 @@ func (r *DNSMasqReconciler) reconcileNormal(ctx context.Context, instance *netwo
 	//Update status with LoadBalancerIPs
 	instance.Status.DNSAddresses = instance.Spec.ExternalEndpoints[0].LoadBalancerIPs
 
+	// Update status with Cluster Addresses
+	poolName := instance.Spec.ExternalEndpoints[0].IPAddressPool
+	serviceName := fmt.Sprintf("%s-%s-%s", dnsmasq.ServiceName, instance.Name, poolName)
+	svc, err := service.GetServiceWithName(ctx, helper, serviceName, instance.Namespace)
+
+	if err != nil {
+		util.LogErrorForObject(helper, err, fmt.Sprintf(
```

Comment:
that message is a duplicate. we already get that as the error message from https://github.com/openstack-k8s-operators/lib-common/blob/main/modules/common/service/service.go#L261 . lets remove that and I think we are good to merge it